### ISSUE TITLE
[BUG] Whoogle cannot be installed with pip

### ISSUE THREAD

**Issue by lol7344** [benbusby/whoogle-search#66]
**Describe the bug**
By typing:
```Text
pip install whoogle-search
```

I get this stack trace:
```Text
    ERROR: Command errored out with exit status 1:
     command: /usr/bin/python3 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-4zn4xd73/whoogle-search/setup.py'"'"'; __file__='"'"'/tmp/pip-install-4zn4xd73/whoogle-search/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-8iry6ov9
         cwd: /tmp/pip-install-4zn4xd73/whoogle-search/
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-4zn4xd73/whoogle-search/setup.py", line 5, in <module>
        requirements = list(open('requirements.txt', 'r'))
    FileNotFoundError: [Errno 2] No such file or directory: 'requirements.txt'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```
I have tried many options, such as:

```Text
pip3 install whoogle-search -r requirements.txt
pip3 install whoogle-search -r "requirements.txt"
pip3 install whoogle-search -r /root/requirements.txt --no-cache-dir
```
None of which worked.
Running as root, or as any other user, does not make any difference.

(the requirements.txt file is set to 777 permissions. Pip shouldn't even need a requirements.txt file!)

**System Info**
Ubuntu 20.04 with Python3.8.2

**Comment by benbusby** [comment]
You're misunderstanding the error. `setup.py` is supposed to read from the repo's requirements.txt file in order to avoid the redundant task of updating requirements in multiple places. This looks like it's caused by the new integration with Travis CI's pypi deployment, I'll switch back to manual for now since it seems something about the repo isn't being set up by them properly.

**Comment by lol7344** [comment]
I have actually understood the error, and I have noticed that indeed my added args are not making any difference, because it is not reading the local requirements.txt. As I said, `pip install` should not need a requirements.txt file.
However, I still wanted to try passing it as an arg, to see if it'd make any difference.

Thanks for such a fast reply, and for all the effort you're putting in this awesome project!

**Comment by benbusby** [comment]
Gotcha. This should be fixed with the v0.1.4 tag I just pushed, but PyPi might not see the update for a bit. Wait a little while before trying to pull the latest version from pip again and make sure the version is the latest. Closing for now, but reopen if the latest version is still giving you issues -- I built the pip package locally first to verify that it works, so barring any weirdness from the automatic deployment through Travis, this should be fixed.
